### PR TITLE
Added buttons for carousel!

### PR DESCRIPTION
**Pull request by samarpanda**
## Next & Previous buttons for carousel!

While trying demos, thought i was missing these buttons. So, let me know if it's a good idea to have it with the demos?

## Updated Screenshoot
screen shot 2016-02-04 at 7 37 57 am

### CODE REVIEW

**Comment by chenglou** [comment]
Good point =)

File: demos/demo4-photo-gallery/Demo.jsx [hunk 1]
```diff
@@ -14,6 +14,18 @@ const Demo = React.createClass({
     this.setState({currPhoto: value});
   },
 
+  clickHandler(e){
+    var btnType = e.target.id === 'nbtn' ? 1 : 0;
```
Comment (chenglou) [inline demos/demo4-photo-gallery/Demo.jsx]: React typically doesn't use IDs for these. Can you create a handler + use this.click.bind(null, ...) or two handlers please?

Comment (samarpanda) [inline demos/demo4-photo-gallery/Demo.jsx]: My bad, updated using `bind`

**Comment by samarpanda** [comment]
@chenglou Are we good to merge? Let me know if we have anything else to address here?

**Comment by chenglou** [comment]
Good to merge! Thanks.

**Comment by chenglou** [comment]
Oops, should have been merged to development instead. I'll correct that.

**Pull request merged**